packages/gatsby-plugin-emotion/src/gatsby-ssr.js

```diff
@@ -1,23 +1,25 @@
-import React from "react"
-import { renderToString } from "react-dom/server"
-import { extractCritical } from "emotion-server"
+const React = require(`react`)
```

Comment:
Should we use an es module here? I prefer to go that way if possible.

It also means some slight refactors to `gatsby-browser.js`

Reply:
I'm cool either way, just wanted to be consistent and some were using CJS so I went with that. Say the word and they'll all be ESM! 🙂

Reply:
Let's do ESM, largely because it _was_ ESM! Bit of a nit, for sure--so thanks for addressing this!

Reply:
The imports were ESM but the exports were CJS. Do you want it that way, or make both imports and exports ESM?

Reply:
Yeah - that's weird!

Consistent! So if using CommonJS, use CommonJS for _all_ the things, if ESM, ESM for _all_ the things.

So e.g. `export const replaceRenderer` is the way to go here!

Reply:
Agreed. Incoming.